apps/meteor/client/views/admin/apps/AppSecurity.tsx

```diff
@@ -75,12 +75,16 @@ const AppSecurity: FC<AppSecurityProps> = ({ privacyPolicySummary, appPermission
 							{t('Policies')}
 						</Box>
 						<Box display='flex' flexDirection='column'>
-							<Box is='a' href={tosLink}>
-								Terms of use
-							</Box>
-							<Box is='a' href={privacyLink}>
-								Privacy policy
-							</Box>
+							{tosLink && (
+								<Box is='a' href={tosLink}>
+									Terms of use
```

Comment:
could you replace this hard text ? use translations keys instead

Reply:
Solved

Reply:
solved where my friend?